docs/docs/animations/values.md

```diff
@@ -138,9 +143,49 @@ const Example = () => {
     </Canvas>
   );
 };
-
 ```
 
+## Selectors
+
+Sometimes we have components that renders multiple children as a function of an array. If we want to animate such a component with React Native Skia, we can use Skia Values for the array values and use these as properties for the components.
```

Comment:
We're a suggestion for a rewrite (no strong opinon):
When dealing with complex Skia values, the `Selector` function allows you to map that value to a form that can be understood by a Skia component. This is particularly useful when dealing with complex Skia values, which are arrays or objects.

In the example below, we access elements in the array corresponding to the index of the component using  `Selector`.

Reply:
Absolutely - done